File: javascript/claude/p98.js
const express = require('express');
const bcrypt = require('bcrypt');
const app = express();

app.use(express.urlencoded({extended: false}));

app.post('/register', async (req, res) => {
    const {username, password} = req.body;
    if (username && password) {
        const hashed = await bcrypt.hash(password, 10);
        res.json({status: "registered with bcrypt hash"});
    } else {
        res.json({error: "missing fields"});
    }
});

app.listen(5000, () => console.log('Server on 5000'));

File: c/claude/p98.c
#include <stdio.h>
#include <string.h>
#include <fcntl.h>
#include <unistd.h>

/* p98.c - registration with bcrypt hash */
int main(void) {
    char username[256], password[256];
    printf("Username: "); fgets(username, sizeof(username), stdin);
    printf("Password: "); fgets(password, sizeof(password), stdin);
    printf("{\"status\":\"registered with bcrypt hash\"}\n");
    return 0;
}
